### PR TITLE
Add Pro Tip mentioning JOSE and the importance of publishing

### PR DESCRIPTION
**Pull request by arm61**

### CODE REVIEW

File: 06-getting-your-class-going-with-jupyter.md [hunk 1]
```diff
@@ -710,6 +710,18 @@ teaching with Jupyter opens new possibilities for educators to engage
 in _open development_ and collaborate with others in producing
 lessons, tutorials, courses, and even books.
 
+```{block2, type='rmdnote'}
+<strong>Pro Tip</strong>
+
+A way to share educational notebooks, gain feedback on them and recieve
```
Comment (labarba) [inline 06-getting-your-class-going-with-jupyter.md]: typo: recieve > receive

File: 06-getting-your-class-going-with-jupyter.md [hunk 2]
```diff
@@ -710,6 +710,18 @@ teaching with Jupyter opens new possibilities for educators to engage
...
+credit for you work is to publish with the
+[Journal of Open Source Education](http://jose.theoj.org). This is a
+peer-reviewed journal aimed at educators developing OERs that use code to
+teach. In addition, to receiving a publication advertising your work, the
```
Comment (labarba) [inline 06-getting-your-class-going-with-jupyter.md]: superfluous comma after "In addition"

**Comment by arm61** [comment]
Sorry for the typos, I am always forgetting to consult grammarly!

**Comment by willingc** [comment]
Thanks @arm61 for the addition :smile:

**Pull request merged**